### DISCUSSION TITLE
Hassio-Addons in homeassistant container

### DISCUSSION THREAD

**Discussion by Quixus** [Poeschl/Hassio-Addons#304] [Q&A] [answered]
Is there a way to use the addons with a containerized version of homeassistant? Simply dropping the files into the ``config/custom_components`` folder does not work.

Thanks in advance and best regards

**Comment by Poeschl** [comment, marked as answer]
There is no way to use any addon in the repository as an component. ;)

An [addon](https://www.home-assistant.io/addons/) adds an additional application to an Home Assistant installation. This in not an integration like everything that goes into the `custom_components` folder.

An [integration](https://www.home-assistant.io/integrations/) is a integration in Home Assistant which enable the control or sensor values of a external thing.

Addons can be only used on a Home Assistant OS or supervised based installation.
